jupyter_firefly_extensions/_version.py

```diff
@@ -1,10 +1,4 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-# Copyright (c) Trey Roby.
-# Distributed under the terms of the Modified BSD License.
-
-
-version_info = (2, 2, 0)
-__version__ = ".".join(map(str, version_info))
-version_short = ".".join(map(str,version_info[0:3]))
+# This file is auto-generated by Hatchling. As such, do not:
+#   - modify
+#   - track in version control e.g. be sure to add to .gitignore
+__version__ = VERSION = '3.0.0'
```

Comment:
We need to have another conversation about what the version number should be. After thinking about it I am leaving to `4` and just skipping 3. There is too much confusion. It will be easier to talk about.

Reply:
Agreed

Reply:
I agree to make it 4. Although I just realised, this change should be in package.json - _version.py is now auto-genrated and gitignored. It still got pushed because it was already existing already - I'll remove it.

Reply:
Done - check package.json